packages/palette/src/elements/Cards/MediumCard.tsx

```diff
@@ -1,10 +1,50 @@
 import React from "react"
+import { Box } from "../Box"
+import { Flex } from "../Flex"
+import { Image } from "../Image"
+import { Spacer } from "../Spacer"
+import { Sans } from "../Typography"
+import { CardTag } from "./CardTag"
 import { MediumCardProps } from "./MediumCard.shared"
 
 /**
  * `MediumCard` is a card with one image one tall image, and text for title and subtitle
  * at the bottom.
  */
-export const MediumCard: React.FC<MediumCardProps> = ({}) => {
-  return null
+export const MediumCard: React.FC<MediumCardProps> = ({
+  image,
+  title,
+  subtitle,
+  tag
+}) => {
+  return (
+    <Box width="280px" height="370px" position="relative">
+      <Flex
+        width="100%"
+        height="100%"
+        position="absolute"
+        top="0"
+        borderRadius="4px"
+        overflow="hidden"
+      >
+        <Image src={image} alt={title} height="100%" width="auto" />
```

Comment:
It's not immediately apparent to me what this would do with a non perfectly sized image? Do we just expect we're always going to pass in a cropped URL?

Reply:
Totally correct @dzucconi - it's currently an issue that we have on a Small card as well. We need to either use `ResponsiveImage` I guess or pass width/height for figure out if width or height should be 100% to cover the space.

The issue is we are in the process of moving from simple URL on the VR to storing and Image model - so it's not exactly clear to me which approach will work better - maybe we can just perfectly size the image with `resize` on graphQl layer.

Reply:
`object-fit: cover` would probably work well here.

Reply:
But yeah a cropped URL as well. I just assume with any lower level component: if it's possible to pass it non-ideal data, then it'll eventually happen. 🤷

Reply:
hmm, `object-fit: cover` definitely fixes the issue, but it's unclear to me whether I should pass that property in through `Image` or if we should switch to a `ResponsiveImage`. I'd advocate merging in this PR as is for now to unblock development and following up with the right solution when we decide on that.